views/partials/index/faq.ejs

```diff
@@ -0,0 +1,51 @@
+<div class="section dummy-section">
+    <div id="faq-start-transition"></div>
+    <!-- start faq section (2 column) -->
+    <div id="faq" class="container">
+        <div class="row">
+            <div class="col-md">
+                <h1 class="section-title">
+                    Frequently Asked Questions
+                </h1>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md">
+                <div class="question-block">
+                    <h5 class="question">What is TAMU Datathon?</h5>
```

Comment:
All headers are Roboto Mono, do we wanna change the headers in FAQs to Work Sans for consistency in the section?